src/applications/edu-benefits/sass/edu-benefits.scss

```diff
@@ -75,16 +75,19 @@
 
       &.institution-name {
         color: #0071BC;
-        font-size: 35;
         font-weight: bold;
-        padding: 10px 0 0 10px;
+        padding: 10px 0 0px 10px;
         text-decoration: underline;
       }
       &.institution-city-state {
-        font-size: 30;
-        font-weight: bold;
         padding: 0 0 10px 10px;
       }
+      &.institution-country {
```

Comment:
Something gross I did here: I'm styling the background of the `span`s here rather than the background of the label. I styled each span separately to get the padding right. 
![screen shot 2018-08-06 at 19 08 45](https://user-images.githubusercontent.com/4998130/43748320-2aef7194-99ac-11e8-89f0-a7ef9c9d2507.png)

The label is kind of messy and big. It's not semantically correct to use block elements or nested elements in a `label` either. 
![screen shot 2018-08-06 at 19 11 10](https://user-images.githubusercontent.com/4998130/43748382-7e2726c2-99ac-11e8-9dc1-aaa3605388e2.png)

![screen shot 2018-08-06 at 19 10 34](https://user-images.githubusercontent.com/4998130/43748376-7706c7d0-99ac-11e8-846c-148a5cdcf6a6.png)

Open to suggestions to make this neater.

Reply:
Could we put all of these spans in a container that handles this padding?